src/pybind/mgr/dashboard/frontend/src/app/ceph/block/rbd-form/rbd-form.component.ts

```diff
@@ -56,20 +56,20 @@ export class RbdFormComponent implements OnInit {
   defaultObjectSize = '4MiB';
```

Comment:
Should this also have a space between the number and unit?

Reply:
Yes, it should. Fixed!